common/constant/default.go

```diff
@@ -46,7 +46,7 @@ const (
 const (
 	DEFAULT_KEY               = "default"
 	PREFIX_DEFAULT_KEY        = "default."
-	DEFAULT_SERVICE_FILTERS   = "echo,token,accesslog,tps,execute,pshutdown"
+	DEFAULT_SERVICE_FILTERS   = "echo,token,accesslog,tps,generic-service,execute,pshutdown"
```

Comment:
Please unify the style，use "generic_service" because https://github.com/apache/dubbo-go/blob/master/filter/impl/hystrix_filter.go#L56.